packages/chakra-ui/src/Widgets/Widgets.ts

```diff
@@ -4,9 +4,9 @@ import CheckboxesWidget from "../CheckboxesWidget/CheckboxesWidget";
 import ColorWidget from "../ColorWidget/ColorWidget";
 import DateWidget from "../DateWidget/DateWidget";
 import DateTimeWidget from "../DateTimeWidget/DateTimeWidget";
-// import EmailWidget from "../EmailWidget/EmailWidget";
```

Comment:
Delete?

Reply:
using this weird that it showed as not being used.